module/hooks/json-changes.mjs

```diff
@@ -0,0 +1,102 @@
+import { DevMode } from '../classes/DevMode.mjs';
+
+Hooks.once('ready', () => {
```

Comment:
instead of setting this up in a hook of its own, `export default` a function which can be used in `foundryvtt-devMode.mjs`'s already existing `ready` hook.

Most of this file can be strictly setting up for the returned function. Only the final bits need to be exported with how you've got this set up. Very nice!

Reply:
```suggestion
```